### PR TITLE
Adding tailwind

### LINKED ISSUE #1: Add tailwind
Add tailwind with typography and daisyui.

https://github.com/svelte-add/tailwindcss

### PR DESCRIPTION
**Pull request by leonardwarner**
Added tailwind
fixes: #1

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -1,2 +1,2 @@
-<h1>Welcome to SvelteKit</h1>
+<h1>Welcome to fortkörning</h1>
 <p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
```
Comment (fabianmossberg) [inline src/routes/+page.svelte]: Remove the link to the sveltekit documentation.

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -1,2 +1,2 @@
-<h1>Welcome to SvelteKit</h1>
+<h1>Welcome to fortkörning</h1>
```
Comment (fabianmossberg) [inline src/routes/+page.svelte]: Content in Swedish?

```suggestion
<h1>Välkommen till fortkörning</h1>

**Review by fabianmossberg — CHANGES_REQUESTED** [review]
See comments.

File: src/routes/+page.svelte (hunk 1 above)
Comment (leonardwarner) [reply src/routes/+page.svelte]: har fixat det

**Review by fabianmossberg — APPROVED** [review]
Looks good!

**Pull request merged**